### PR TITLE
Basic dockerfile and instructions to execute

### PR DESCRIPTION
**Pull request by sadekbaroudi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,14 @@
 # rome
 A new build system for SugarCRM
+
+# docker
+
+Build the docker image:
+```
+docker build -t registry.sugarcrm.net/rome:1.0.0 .
```
Comment (jwhitcraft) [inline README.md]: lets put it here,

`registry.sugarcrm.net/engineering/rome:latest`

and this is v2.0.0Alpha for now since 1.0 exists in the Mango repo.

**Review by jwhitcraft — CHANGES_REQUESTED** [review]
See Comments :P

File: README.md (hunk 1 above)
Comment (sadekbaroudi) [reply README.md]: Updated to latest!

**Comment by sadekbaroudi** [comment]
Resolved!

**Review by jwhitcraft — APPROVED** [review]
Approved :shipit: :goat:

**Pull request merged**